api/src/main/java/org/pmiops/workbench/api/ConceptsController.java

```diff
@@ -65,16 +69,37 @@
             .vocabularyId(concept.getVocabularyId())
             .conceptSynonyms(concept.getSynonyms().stream().map(ConceptSynonym::getConceptSynonymName).collect(Collectors.toList()));
 
+  static final Function<DbDomain, DomainInfo> TO_CLIENT_DOMAIN_INFO =
+      (dbDomain) -> new DomainInfo()
+          .domain(Domain.fromValue(dbDomain.getDomainId().toUpperCase()))
+          .name(dbDomain.getDomainDisplay())
+          .description(dbDomain.getDomainDesc())
+          .allConceptCount(dbDomain.getAllConceptCount())
+          .standardConceptCount(dbDomain.getStandardConceptCount())
+          .participantCount(dbDomain.getParticipantCount());
+
   @Autowired
   ConceptSynonymDao conceptSynonymDao;
 
   @Autowired
-  public ConceptsController(ConceptService conceptService, WorkspaceService workspaceService,ConceptSynonymDao conceptSynonymDao) {
+  public ConceptsController(ConceptService conceptService, WorkspaceService workspaceService,
+                            ConceptSynonymDao conceptSynonymDao, DbDomainDao dbDomainDao) {
     this.conceptService = conceptService;
     this.workspaceService = workspaceService;
     this.conceptSynonymDao = conceptSynonymDao;
+    this.dbDomainDao = dbDomainDao;
   }
 
+  @Override
+  public ResponseEntity<DomainInfoResponse> getDomainInfo(String workspaceNamespace,
+      String workspaceId) {
```

Comment:
Concept sets are scoped to a workspace. You can think of them existing independent from workspaces in future (much like cohort definitions), but for now they only reside inside a workspace.

That said, this method just has to do with domains, not concept sets. The reason to have it be tied to a workspace is so we can do workspace-specific things in future (as mentioned above.)

We *could* switch this to a separate method that just takes the CDR version ID instead of the workspace, and verifies you have access to that CDR version instead here. But then we won't be able to do workspace-specific stuff in future without switching back.